modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -103,10 +113,11 @@ const pAssertMenuPresence = async (
   container: SugarElement<HTMLElement>,
   useMenuOrToolbar: 'toolbar' | 'menuitem'
 ): Promise<void> => {
+  const title = useMenuOrToolbar === 'toolbar' ? menuTitleToToolbarBarMap[menuTitle] : menuTitle;
```

Comment:
This is the sort of reason why I wanted to apply the `data-mce-name` concept everywhere and never use `aria-label` in tests 🙃 it would make the tests so much easier and more consistent

But there isn't scope for that now.